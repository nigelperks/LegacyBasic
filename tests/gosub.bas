1000 GOSUB 2000
1010 GOSUB 2000
1020 GOSUB 2000
1030 GOSUB 2200
1040 END
2000 PRINT "IT IS SO ROUTINE"
2010 RETURN
2200 PRINT "OUTER BEGIN"
2210 GOSUB 2300
2220 PRINT "OUTER END"
2230 RETURN
2300 PRINT "INNER"
2310 RETURN
