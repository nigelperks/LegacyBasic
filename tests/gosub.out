IT IS SO ROUTINE
IT IS SO ROUTINE
IT IS SO ROUTINE
OUTER BEGIN
INNER
OUTER END
